README.md

```diff
@@ -121,18 +121,18 @@ You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and
 6. Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin supports apps that are published in the Zoom Marketplace. 
 7. Select **Create**.
 
-![Create account level app](https://github.com/mattermost/mattermost-plugin-zoom/assets/74422101/a7af4594-9b22-43ea-9dd6-c4bf7ecbd794")
+![Create account level app](https://github.com/mattermost/mattermost-plugin-zoom/assets/74422101/a7af4594-9b22-43ea-9dd6-c4bf7ecbd794)
```

Comment:
@cwarnermm FYI I'm noticing we're using the `assets` directory. I know we discussed this some time before but wondering where we stand with relocating docs-only things to a `docs` folder and not the `assets` folder

Reply:
@mickmister - We definitely want to move docs-only components to a ``/docs`` directory over using the ``/assets`` directory. Some cleanup has taken place in this area via Hacktoberfest, but isn't complete.